bin/radical-pilot-create-static-ve

```diff
@@ -112,19 +112,22 @@ esac
 
 PYTHON=python3
 
+if test -z "$VERSION"
+then
+    VERSION=$($PYTHON -V | cut -f 2 -d ' ' | cut -f 1,2 -d '.')
+fi
+if test -z "$VERSION"
+then
+    VERSION='3.8'
+fi
+
+# we only ever use major and minor version numbers, and will ignore subminor
+VERSION=$(echo $VERSION | cut -f 1,2 -d '.')
```

Comment:
maybe move bottom line into if-statement?
```
if test -z "$VERSION"
then
    VERSION='3.8'
else
    # we only ever use major and minor version numbers, and will ignore subminor
    VERSION=$(echo $VERSION | cut -f 1,2 -d '.')
fi
```
but could be skipped

Reply:
like to check if subminor is specified in the first place?  Counting characters is cumbersome in shell, but also, the `cut` command also works if version is empty, major only or major.minor, so this should be safe.